### PR TITLE
Fix @threlte/extras text and postprocessing outline docs examples

### PR DESCRIPTION
**Pull request by michealparks**
😎 🎉

### CODE REVIEW

File: apps/docs/src/content/reference/core/use-threlte.mdx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const {
 
 ### Frameloop
 
-If the [frameloop is set to `'demand'`]/docs/reference/core/canvas) and you are manually editing objects or materials, be sure to invalidate the current frame to request a rerender:
+If the [frameloop is set to `'demand'`](/docs/reference/core/canvas) and you are manually editing objects or materials, be sure to invalidate the current frame to request a rerender:
```
Comment (michealparks) [inline apps/docs/src/content/reference/core/use-threlte.mdx:57]: Restored missing `(`

File: apps/docs/src/examples/extras/text/App.svelte [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 <div use:action />
 
-<div class="w-full h-full -z-10 relative bg-orange-500/20">
+<div class="w-full h-full relative bg-orange-500/20">
```
Comment (michealparks) [inline apps/docs/src/examples/extras/text/App.svelte:29]: Negative z-index was hiding the canvas

File: apps/docs/src/examples/postprocessing/outline/CustomRenderer.svelte [hunk 3]
```diff
@@ -1,40 +1,42 @@
...
-			xRay: true,
-			blur: true
-		});
-		if (selectedMesh !== undefined) {
-			outlineEffect.selection.set([selectedMesh]);
-		}
-		composer.addPass(new EffectPass(camera, outlineEffect));
-	};
+  $: setupEffectComposer($camera, selectedMesh)
+  $: composer.setSize($size.width, $size.height)
```
Comment (michealparks) [inline apps/docs/src/examples/postprocessing/outline/CustomRenderer.svelte:37]: Not setting the composer size was resulting in a zero size framebuffer

**Review by grischaerbe — APPROVED** [review]
smooth 😎

**Pull request merged**